RobotCode/snobot2018/src/org/snobot/Snobot2018.java

```diff
@@ -27,10 +31,13 @@ public void robotInit()
 
         // Joystick
         Joystick driverJoystickRaw = new Joystick(PortMappings2018.sDRIVER_JOYSTICK_PORT);
-
         SnobotDriveXbaxJoystick driverJoystick = new SnobotDriveXbaxJoystick(driverJoystickRaw, logger);
         addModule(driverJoystick);
 
+        Joystick elevatorJoystickRaw = new Joystick(PortMappings2018.sELEVATOR_JOYSTICK_PORT);
+        SnobotDriveOperatorJoystick OperatorJoystick = new SnobotDriveOperatorJoystick(elevatorJoystickRaw, logger);
+        addModule(OperatorJoystick);
+
         // DriveTrain
         SpeedController driveLeftMotor = new VictorSP(PortMappings2018.sDRIVE_PWM_LEFT_A_PORT);
```

Comment:
@team174-calabria We will need to figure out factory/dependency injection sooner rather than later.  The real robot will use TalonSRX and built in encoders rather than <any other SC> and encoders